src/http/output/error/ConvertingErrorHandler.ts

```diff
@@ -25,30 +26,34 @@ type PreparedArguments = {
  */
 export class ConvertingErrorHandler extends ErrorHandler {
   private readonly converter: RepresentationConverter;
+  private readonly preferenceParser: PreferenceParser;
   private readonly showStackTrace: boolean;
 
-  public constructor(converter: RepresentationConverter, showStackTrace = false) {
+  public constructor(converter: RepresentationConverter, preferenceParser: PreferenceParser, showStackTrace = false) {
```

Comment:
I wonder if we want a full `BasicRequestParser` for forward compatibility (and we cache).